syntax = "proto3";
import "google/protobuf/empty.proto";


// clientProxy send it to requestServer
message requestQuery{
    int32 cid = 1; // client id
    int32 qid = 2; //query id
    string db_name = 3; // database name
    string table_name = 4;
    string column_name =5; // database column name
    string op = 6; // operation
    string ipaddress = 7;
}

service ParseServerService {
  //  requestServer provides the interface, clientProxy remotes the call
  rpc RequestParsing (requestQuery) returns (google.protobuf.Empty);

}